Projekt Zaliczeniowy/Projekt_zaliczeniowy.Rmd

```diff
@@ -0,0 +1,293 @@
+as.tibble(rozwody)
+rozwody %>% mutate(rozwody_ogolem = `ogółem`) -> nowerozwody
+as.tibble(nowerozwody)
+
+skolaryzacja  <- readxl::read_xlsx('Skolaryzacja.xlsx', sheet = 2)
+as.tibble(skolaryzacja)
+
+ludnosc  <- readxl::read_xlsx('Ludnosc.xlsx', sheet = 2)
+as.tibble(ludnosc)
+ludnosc %>% mutate(ludnosc_ogolem = `ogółem`) -> noweludnosc
+as.tibble(noweludnosc)
+
+gestosc  <- readxl::read_xlsx('Gestosc.xlsx', sheet = 2)
+as.tibble(gestosc)
+gestosc %>% mutate(gestosc_ogolem = `ludność na 1 km2`) -> nowegestosc
+as.tibble(nowegestosc)
+
+# Złączenia tabel
+tab1 <- powiaty %>% left_join(nowemalzenstwa,by = c('JPT_KOD_JE' = 'Kod'))
+tab2 <- tab1 %>% left_join(nowerozwody,by = c('JPT_KOD_JE' = 'Kod'))
+tab3 <- tab2 %>% left_join(skolaryzacja, by = c('JPT_KOD_JE' = 'Kod'))
+tab4 <- tab3 %>% left_join(noweludnosc, by = c('JPT_KOD_JE' = 'Kod'))
+tab5 <- tab4 %>% left_join(nowegestosc, by = c('JPT_KOD_JE' = 'Kod'))
+polaczona <- tab5 %>% left_join(nowewynagrodzenia,by = c('JPT_KOD_JE' = 'Kod'))
+
+# Zamiana typów danych
+polaczona$ogółem.x <- as.integer(polaczona$ogółem.x)
+polaczona$malzenstwa_ogolem <- as.integer(polaczona$malzenstwa_ogolem)
+polaczona$ogółem.y <- as.integer(polaczona$ogółem.y)
+polaczona$rozwody_ogolem <- as.integer(polaczona$rozwody_ogolem)
+polaczona$`brutto szkoły podstawowe` <- as.integer(polaczona$`brutto szkoły podstawowe`)
+polaczona$`netto szkoły podstawowe` <- as.integer(polaczona$`netto szkoły podstawowe`)
+polaczona$`brutto gimnazja` <- as.integer(polaczona$`brutto gimnazja`)
+polaczona$`netto gimnazja` <- as.integer(polaczona$`netto gimnazja`)
+polaczona$ludnosc_ogolem <- as.integer(polaczona$ludnosc_ogolem)
+polaczona$gestosc_ogolem <- as.integer(polaczona$gestosc_ogolem)
+```
+
+## Wstęp
+
+W poniższym projekcie przedstawiamy poziom wynagrodzeń, współczynnik skolaryzacji oraz sytuację matrymonialną i kolerację między nimi.
+Wszystkie przedstawione poniżej dane dotyczączą roku 2018.
+
+## Ludność w Polsce
+
+Bilanse liczby i struktury ludności w gminach imiennie opracowane w oparciu o wyniki Narodowych Spisów Powszechnych z uwzględnieniem zmian spowodowanych ruchem naturalnym (urodzenia i zgony), migracjami ludności (na pobyt stały i czasowy) oraz przemieszczeniami związanymi ze zmianami administracyjnymi.
+
+```{r wykres1, echo=FALSE}
+
+w1 <- ggplot(polaczona, aes(fill = polaczona$ludnosc_ogolem)) +
+  geom_sf() +
+  ggtitle("Liczba") +
+  theme(legend.title=element_blank())
+
+w2 <- ggplot(polaczona, aes(fill = polaczona$gestosc_ogolem)) +
+  geom_sf() +
+  ggtitle("Gęstość na 1km2") +
+  theme(legend.title=element_blank())
+
+grid.arrange(arrangeGrob(w1,w2, ncol=2, nrow=1),
+              top = "Liczba ludności", heights=c(10,10)) 
+```
+
+Na zestawieniu poniżej powiaty z największą liczbą ludności oraz powiaty z największą gęstością zaludnienia. Widać, że są to miasta. Co ciekawe liczba ludności nie koniecznie wskazuje nam na gęstość zaludnienia w danym powiecie.

```

Comment:
Tekst w pliku Rmd dobrze robić tak, aby każde zdanie było w nowej linijce. W skompilowanym piku jeśli jest to tylko przesunięcie do następnej linijki, to zostanie zachowana ciągłość, ale dzięki takiemu manewrowi dowolna zmiana w jednym zdaniu pokaże nam w git w różnicy pomiędzy plikami, w którym dokładnie zdaniu (w której linijce) nastąpiła zmiana i nie musimy przeszukiwać całego długiego akapitu.